packages/wear-os/index.ts

```diff
@@ -8,10 +8,27 @@ export * from './tasks';
 export * from './watch';
 
 import { PluginLoader } from '@awarns/core/common';
-import { wearosSensors, allSensors, defaultConfig, WearosSensorsConfig as WSC } from 'nativescript-wearos-sensors';
+import { wearosSensors } from 'nativescript-wearos-sensors';
 import { WatchSensorsProvider } from './internal/provider';
+import { toSensorType, WatchSensor } from './internal/watch-sensor';
 
-export type WearOSPluginConfig = WSC;
+export interface WearOSPluginConfig {
+  sensors?: WatchSensor[];
+  disablePlainMessaging?: boolean;
+  disableWearCommands?: boolean;
+}
+
+export const allSensors = [
+  WatchSensor.ACCELEROMETER,
+  WatchSensor.GYROSCOPE,
+  WatchSensor.MAGNETOMETER,
+  WatchSensor.HEART_RATE,
+  WatchSensor.GEOLOCATION,
+];
+
+export const defaultConfig = {
+  sensors: allSensors,
+};
```

Comment:
I see in fact that the default config disables the optional features, so indeed, the docs should be updated to reflect this

Reply:
Not really, all is enabled by default. The logic tests `!disablePlainMessaging` to configure things, so if no value provided then `!undefined === true`. But yeah, I can change the configuration to only enable by default all the sensors. I will also change the name of the config properties to `enable...`